### PR TITLE
[ISSUE #9769] Fix config auth failed.

### PR DESCRIPTION
**Pull request by YunWZ**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

nacos-client login failed.(In ConfigTransportClient).

related issue #9734 , #9769 .

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: client/src/main/java/com/alibaba/nacos/client/config/impl/ServerListManager.java [hunk 1]
```diff
@@ -198,17 +195,7 @@ public ServerListManager(NacosClientProperties properties) throws NacosException
             StringTokenizer serverAddrsTokens = new StringTokenizer(this.serverAddrsStr, ",;");
             while (serverAddrsTokens.hasMoreTokens()) {
                 String serverAddr = serverAddrsTokens.nextToken().trim();
-                if (serverAddr.startsWith(HTTP_PREFIX) || serverAddr.startsWith(HTTPS_PREFIX)) {
```
Comment (The-Gamer-01) [inline client/src/main/java/com/alibaba/nacos/client/config/impl/ServerListManager.java:201]: 此处为什么可以省略此判断？我好像并未发现其他地方对serverAddr做了新的处理。

File: client/src/main/java/com/alibaba/nacos/client/config/impl/ServerListManager.java [hunk 2]
```diff
@@ -388,14 +375,7 @@ private void updateIfChanged(List<String> newList) {
             return;
         }
         
-        List<String> newServerAddrList = new ArrayList<>();
-        for (String server : newList) {
```
Comment (The-Gamer-01) [inline client/src/main/java/com/alibaba/nacos/client/config/impl/ServerListManager.java:392]: 此处为什么可以省略前缀判断？并未发现其他对server列表的处理。

**Review by The-Gamer-01 — COMMENTED** [review]
CI/CD未通过，希望先在本地修改通过后再进行PR。

File: client/src/main/java/com/alibaba/nacos/client/config/impl/ServerListManager.java (hunk 2 above)
Comment (YunWZ) [reply client/src/main/java/com/alibaba/nacos/client/config/impl/ServerListManager.java:392]: see about #9734

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**